src/ray/common/metrics.h

```diff
@@ -0,0 +1,43 @@
+// Copyright 2025 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include "ray/stats/metric.h"
+
+namespace ray {
+
+inline ray::stats::Gauge GetActorMetric() {
+  /// Tracks actors by state, including pending, running, and idle actors.
+  ///
+  /// To avoid metric collection conflicts between components reporting on the same actor,
```

Comment:
I'm a little confused by this.  Right now we have component labels for things like raylet and gcs and we have a Name label for a metric name.  Now we're adding a source label.  Why would two components necessarily conflict in this set up?  Are they within the same component?